pyfr/solvers/euler/inters.py

```diff
@@ -21,9 +46,9 @@ def __init__(self, *args, **kwargs):
         )
 
 
-class EulerMPIInters(BaseAdvectionMPIInters):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class EulerMPIInters(FluidMPIIntersMixin, BaseAdvectionMPIInters):
+    def __init__(self, be, lhs, rhsrank, rallocs, elemap, cfg):
```

Comment:
Do we need the explicit args here?

Reply:
NS has explicit args but Euler does not. Which one do you want to go with?

Reply:
Use implicit args unless you need a substantial number of the arguments in the function (in which case explicit is cleaner/the only option).

Reply:
Do you want to switch the NS ones then? Most of them don't use any of the args